k8s/spark-operator/manifest/application.yaml.template

```diff
@@ -7,7 +7,7 @@ metadata:
-    marketplace.cloud.google.com/deploy-info: '{partner_id: "click-to-deploy-containers", product_id: "elasticsearch", partner_name: "Google Click to Deploy"}'
+    marketplace.cloud.google.com/deploy-info: '{partner_id: "click-to-deploy-containers", product_id: "spark", partner_name: "Google Click to Deploy"}'
```

Comment:
Thanks for catching this! `spark-operator`--this should match whatever on marketplace based on its URL: http://console.cloud.google.com/marketplace/details/google/spark-operator

Reply:
Done.